src/drivers/distance_sensor/lightware_laser_i2c/parameters.c

```diff
@@ -54,7 +54,7 @@ PARAM_DEFINE_INT32(SENS_EN_SF1XX, 0);
  *
  * @value 0 Disabled
  * @value 1 Enabled
- * @value 2 Disabled during VTOL fast forward flight
+ * @value 2 Enabled in VTOL MC mode, listen to request from system in FW mode
```

Comment:
Why only listen to request from system in FW mode and not always? What are the drawbacks?

Reply:
Do we even need to keep the VTOL dependency? Would it be enough to only link it to the isLanding state?

Reply:
Yes, that is a valid point, i mainly did it for backward compatibility. General idea was that it is always enabled for MC mode, irrespective what the request is. For VTOL this is mainly for takeoff and landing. L conditions, is this also ok for MCs? better approach would be to get rid of the vehicle status condition in the driver and let navigator/commander/estimator choose, if it should be enabled of not.
As @sfuhrer mentioned, it would be even better if the estimator can request enabling the distance sensor if it helps with optical flow. But that would make the PR even bigger, so i would suggest a follow up PR to take care of that.

Reply:
So is the current approach ok? We currently leaving it as is with auto enabled in VTOL mode and shuld change that at some point to make it non VTOL related?